protected/humhub/modules/user/services/AuthClientService.php

```diff
@@ -105,6 +106,18 @@ public function updateUser(User $user = null): bool
             if (count($user->profile->getDirtyAttributes()) !== 0 && !$user->profile->save()) {
                 Yii::warning('Could not update user profile (' . $user->id . '). Error: '
                     . VarDumper::dumpAsString($user->profile->getErrors()), 'user');
+
+                return false;
+            }
+        } elseif ($isNoLocalAuth && $user->status === User::STATUS_SOFT_DELETED) {
+            // Change status from soft deleted to enabled
+            // if user logged in with no local auth method
+            $user->setAttribute('status', User::STATUS_ENABLED);
```

Comment:
It should not be possible to recover "Soft Deleted" user accounts. Instead, a new account should be created for the user.

Presumably the cause of the original bug report was that a "soft deleted" user tried to log in before the ActiveJob run. I.e. the current behavior should be correct, sorry for the confusion.

Reply:
I removed it